### ISSUE TITLE
Add keyboard action to the Submit button

### ISSUE THREAD

**Issue by tminussi** [tminussi/hacktoberfest-2018-checker#9]
Currently, we have to click on the "submit" button in the index.html file to perform the query.

Add keyboard actions to it (let's say, if a user presses the ENTER key, it should also run the action) so it becomes more user friendly

**Comment by SpaceLenore** [comment]
I can work on this, unfortunately I have some issues setting up the development environment (tried both with and without vscode) luckily this issue can be fixed without the backend.

**Comment by tminussi** [comment]
thanks @SpaceLenore. Can you please let me know what went wrong when setting up the environment? I can tell you how to fix it then you can submit a PR for the README with the changes.

**Comment by SpaceLenore** [comment]
First I ran `npm install` in the server directory then I installed serverless and serverless-offline globally but when I tried to start i got this error
```
$ sudo serverless offline start
 
  Serverless Error ---------------------------------------
 
  Serverless command "offline" not found. Did you mean "config"? Run "serverless help" for a list of all available commands.
 
  Get Support --------------------------------------------
     Docs:          docs.serverless.com
     Bugs:          github.com/serverless/serverless/issues
     Issues:        forum.serverless.com
 
  Your Environment Information -----------------------------
     OS:                     linux
     Node Version:           9.11.2
     Serverless Version:     1.32.0
```

after that I tried to use vscode and when I start the debugger I get a message box saying
`Attribute 'program' does not exist ('/home/SpaceLenore/Documents/Git/hacktoberfest-2018-checker/node_modules/serverless/bin/server')`

**Comment by hovoodd** [comment]
@SpaceLenore I have the same issue, but I just installed serverless packages manually and launched the project from the command line

**Comment by tminussi** [comment]
Thank you @hovoodd and @SpaceLenore  for your inputs!

I have fixed it accordingly. When moving projects around, I ended up leaving the .vscode directory one level up. You shouldn't have any issues now.

I am sorry for any inconvenience and thanks for bringing it up!

**Comment by tminussi** [comment]
Fixed by @SpaceLenore on PR #12
